### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by lishaduck**
Unfortunately, one leaked into the public api, necessitating a major.

### CODE REVIEW

**Comment by lishaduck** [comment]
Docusaurus claims I broke a link, but I'm 99% sure it's wrong.

**Comment by lishaduck** [comment]
Oh, nope, it's right. I'll fix it!

**Comment by AndreaPontrandolfo** [comment]
Oh gosh, i have no idea how this did slip trough haha

**Comment by lishaduck** [comment]
> Oh gosh, i have no idea how this did slip trough haha

It's been like this the whole time, but I never got around to complaining 😆

**Comment by AndreaPontrandolfo** [comment]
I had Megalinter setup on CI but i had to disable it because it was too much noise.

**Comment by lishaduck** [comment]
> I had Megalinter setup on CI but i had to disable it because it was too much noise.

Yeah. I've found running streetsidesoftware/cspell-action to be good at catching it.

**Comment by AndreaPontrandolfo** [comment]
Yeah Cspell is integrated into Megalinter. 
But again, it was too much work to keep up with.

>  I couldn't get megalinter to play nice with my ESLint config.

I simply disabled the Eslint part of Megalinter, as obviously i didn't need it.

**Comment by lishaduck** [comment]
Oh, I guess once we get typescript-eslint@8, then we could remove `pathsOverrides` in favor of using the `projectService`.

File: .prettierignore [hunk 1]
```diff
@@ -39,4 +39,7 @@ dist
 
 megalinter-reports/
 
-pnpm-lock.yaml
\ No newline at end of file
+pnpm-lock.yaml
+
+# MDX v3
+*.mdx
```
Comment (AndreaPontrandolfo) [inline .prettierignore:45]: I'm ok with this.
Let's track https://github.com/prettier/prettier/issues/12209 until it's fixed.

**Review by AndreaPontrandolfo — APPROVED** [review]

**Comment by AndreaPontrandolfo** [comment]
> Oh, and I guess once we get typescript-eslint@8, then we could remove `pathsOverrides` in favor of using the `projectService`.

What do you mean?

**Comment by lishaduck** [comment]
> > Oh, and I guess once we get typescript-eslint@8, then we could remove `pathsOverrides` in favor of using the `projectService`.
> 
> What do you mean?

https://typescript-eslint.io/blog/announcing-typescript-eslint-v8-beta/#project-service

**Comment by AndreaPontrandolfo** [comment]
> > > Oh, and I guess once we get typescript-eslint@8, then we could remove `pathsOverrides` in favor of using the `projectService`.
> > 
> > 
> > What do you mean?
> 
> https://typescript-eslint.io/blog/announcing-typescript-eslint-v8-beta/#project-service

I'll start a discussion so we can merge this in the meantime.

**Comment by AndreaPontrandolfo** [comment]
Discussion started here: https://github.com/AndreaPontrandolfo/sheriff/discussions/160

**Pull request merged**